src/services/administration/store/user-page-store.ts

```diff
@@ -0,0 +1,76 @@
+import { computed, reactive } from 'vue';
+
+import { defineStore } from 'pinia';
+
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
+import type { Query } from '@cloudforet/core-lib/space-connector/type';
+
+import { store } from '@/store';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
+import { calculateTime } from '@/services/administration/iam/user/lib/helper';
+import type { User, UserType } from '@/services/administration/iam/user/type';
+import { USER_TYPE } from '@/services/administration/iam/user/type';
+
+
+export const useUserPageStore = defineStore('user-page', () => {
+    const state = reactive({
+        loading: false,
+        users: [] as User[],
+        totalCount: 0,
+        selectedIndices: [],
+        visibleManagementModal: false,
+        visibleCreateModal: false,
+        visibleUpdateModal: false,
+    });
+
+    const getters = reactive({
+        timezone: computed(() => store.state.user.timezone || 'UTC'),
+        selectedUsers: computed(() => {
+            const users = [] as User[];
+            state.selectedIndices.map((d) => users.push(state.users[d]));
+            return users;
+        }) || [],
+    });
+
+    /* Actions */
+    const _getArrayWithNotDuplicatedItem = (array) => [...new Set(array)];
```

Comment:
Question: Last time I used `_`, the warning log came up on the console, is it okay this time??

Reply:
Personally, I really like the underbar prefix in the internal function!

Reply:
It seems to give such a warning when using a `_state` in a template.
There is no warning yet for using within a script!